src/nodes/toJS.ts

```diff
@@ -10,6 +14,8 @@ export interface AnchorData {
 
 export interface ToJSContext {
   anchors: Map<Node, AnchorData>
+  /** Cached anchor and allias nodes in the order they occurred in the document */
+  anchorAndAliasNodes?: (Alias | Scalar | YAMLMap | YAMLSeq)[]
```

Comment:
Why is this optional? It might also be better to name it to correspond to its use, i.e. something like `aliasResolveCache` or similar. Not sure though.

Reply:
Renamed, at first I made it optional so that the change is non-api braking, but in the new version I use it to figure out if cache was already built or not. If it is undefined cache needs to be built.